scrapers/base.py

```diff
@@ -62,14 +62,16 @@ def is_container_build(self, build_info):
         except (ValueError, TypeError):
             extra_json = {}
 
-        # Checking a heuristic for determining if a build is a container build since, currently
+        # Checking heuristics for determining if a build is a container build since, currently
         # there is no definitive way to do it.
         if extra_json and (extra_json.get('container_koji_build_id') or
                            extra_json.get('container_koji_task_id')):
             return True
-        # Checking another heuristic for determining if a build is a container build since
-        # currently there is no definitive way to do it.
-        elif (package_name.endswith('-container') or package_name.endswith('-docker')):
+        elif extra_json.get("image"):
```

Comment:
After talking with Luiz, I think we can change this to:
```python
elif extra_json.get('image') and (package_name.endswith('-container') or package_name.endswith('-docker')):
    return True
```

Then we can remove the rest of the if statements.

Reply:
I think we should keep the one which checks for `container_koji_build_id`. Just to be sure